pkg/karmadactl/token/token.go

```diff
@@ -300,11 +300,11 @@ func (o *CommandTokenOptions) runDeleteTokens(out io.Writer, client kubeclient.I
 		}
 
 		tokenSecretName := bootstraputil.BootstrapTokenSecretName(tokenID)
-		klog.V(1).Infof("[token] deleting token %q", tokenID)
+		klog.V(1).Infof("[token] deleting secret %s", tokenSecretName)
```

Comment:
refer to https://github.com/karmada-io/karmada/blob/774a2b13665380deedd18bcb319defa9aa5743be/pkg/karmadactl/token/token.go#L56-L57, the tokenID in fact is a public id without sensitive information. So this pr is just to silence some security scanning tools.

Reply:
> 2. token id is also a suspicion that it is considered sensitive information, so don't print it if you can avoid it.

TokenId does not contain sensitive information, in fact, the `tokenSecretName` itself contains the `token id`.

Reply:
Just in our case, this is bootstrap token id and does not contain sensitive information, but does not represent all variables named `tokenid`. Then, for some black box scanning tools, it is easy to be mistakenly scanned. So when it comes to printing logs, we can avoid them if we can.